packages/hierarchies-react/src/presentation-hierarchies-react/LocalizationContext.tsx

```diff
@@ -0,0 +1,59 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { createContext, PropsWithChildren, useContext, useEffect, useState } from "react";
+
+/** @internal */
+export interface Localization {
+  loading: string;
+  filterHierarchyLevel: string;
+  clearHierarchyLevelFilter: string;
+  noFilteredChildren: string;
+  resultLimitExceeded: string;
+  resultLimitExceededWithFiltering: string;
+  increaseHierarchyLimit: string;
+  increaseHierarchyLimitWithFiltering: string;
+}
+
+/** @internal */
+export interface LocalizationContext {
+  localization: Localization;
+}
+
+const defaultLocalization: Localization = {
```

Comment:
For someone localizing these string it could be inconvenient to find default values. Maybe also add them as a comment for each attribute of `Localization` type?